python/tests/simplify.py

```diff
@@ -192,13 +192,16 @@ def record_edge(self, left, right, parent, child):
             left = X[left_index]
             right = X[right_index]
         if child not in self.edge_buffer:
-            self.edge_buffer[child] = [tskit.Edge(left, right, parent, child)]
+            self.edge_buffer[child] = [tskit.Edge(
+                left=left, right=right, parent=parent, child=child, id_=None)]
```

Comment:
``id_`` defaults to None, so no point in changing this, right?

Reply:
Good point. I had forgotten about the default